meta-iot2000-example/recipes-support/swupdate/swupdate/defconfig

```diff
@@ -45,31 +47,48 @@ CONFIG_BOOTLOADER_EBG=y
 # CONFIG_UBOOT is not set
 # CONFIG_BOOTLOADER_NONE is not set
 # CONFIG_BOOTLOADER_GRUB is not set
-# CONFIG_DOWNLOAD is not set
-# CONFIG_HASH_VERIFY is not set
+CONFIG_DOWNLOAD=y
+CONFIG_CHANNEL_CURL=y
+CONFIG_HASH_VERIFY=y
 # CONFIG_SIGNED_IMAGES is not set
 # CONFIG_ENCRYPTED_IMAGES is not set
 # CONFIG_SURICATTA is not set
-# CONFIG_WEBSERVER is not set
 
 #
-# Archival Features
+# Features
+#
+# CONFIG_SURICATTA_SSL is not set
+# CONFIG_SURICATTA_STATE_CHOICE_NONE is not set
+# CONFIG_SURICATTA_STATE_CHOICE_BOOTLOADER is not set
+# CONFIG_SURICATTA_STATE_BOOTLOADER is not set
+
 #
+# Server
+#
+# CONFIG_SURICATTA_HAWKBIT is not set
+# CONFIG_WEBSERVER is not set
 CONFIG_GUNZIP=y
 
 #
 # Parser Features
 #
 CONFIG_LIBCONFIG=y
 CONFIG_PARSERROOT=""
-# CONFIG_JSON is not set
+CONFIG_JSON=y
+CONFIG_LUAEXTERNAL=y
+# CONFIG_SETEXTPARSERNAME is not set
 # CONFIG_SETSWDESCRIPTION is not set
 
 #
 # Image Handlers
 #
 CONFIG_RAW=y
-# CONFIG_SHELLSCRIPTHANDLER is not set
+CONFIG_LUASCRIPTHANDLER=y
+CONFIG_SHELLSCRIPTHANDLER=y
```

Comment:
Well, we could provide a *minimal* `.config`, serving also as documentation for the minimal features required.

Reply:
The .config is the minimal config except the LUA handler. To reduce the amount of commits, I included the config that was most likely to be useful. No backends were anticipated, but without round-robin, updates are ugly to maintain. Round-robin should be a default feature IMO.

Reply:
Is the `shellhandler` required? There are others that are not strictly required, aren't they?

Reply:
Okay, I give it a 2nd run and minimize it.